analyzers/tests/SonarAnalyzer.UnitTest/TestCases/MemberOverrideCallsBaseMember.CSharp10.cs

```diff
@@ -28,9 +30,46 @@ record Derived : Base
     public override sealed int MyProperty5 { get => base.MyProperty5; init => base.MyProperty5 = value; }
     public override int MyProperty6 { get => base.MyProperty6; init => base.MyProperty6 = value; } // Error [CS0154, CS0154] The property or indexer 'property' cannot be used in this context because it lacks the get accessor
     public override int MyProperty7 { get => base.MyProperty7; init => base.MyProperty7 = value; } // Error [CS0200, CS0200] Property or indexer 'property' cannot be assigned to -- it is read only
+    public override int MyProperty8 { init => base.MyProperty8 = value; } // Noncompliant
+    public override int MyProperty9 { get => base.MyProperty9; } // Noncompliant
     public sealed int MyProperty10 { get => base.MyProperty10; init => base.MyProperty10 = value; } // Error [CS0238] 'member' cannot be sealed because it is not an override
     public override int MyProperty11 { get => base.MyProperty11; init => backingField = value; }
     public override int MyProperty12 { get => base.MyProperty5; init => base.MyProperty5 = value; }
     public override int MyProperty13 { get; init => base.MyProperty13 = value; } // Error [CS0501] 'member function' must declare a body because it is not marked abstract, extern, or partial
     public override int MyProperty14 { get => base.MyProperty14; init => base.MyProperty14 = value; } // Error [CS0115, CS0117, CS0117] no suitable method found to override, 'Base' does not contain a definition for 'MyProperty14'
 }
+
+namespace CompilerGeneratedMethods
+{
+    record Base
+    {
+        public override string ToString() => "Some custom ToString";
```

Comment:
hmm I see we only have tests with properties for records - how about adding also a `Method()` here , to test that overriding it and simply calling base will be `Noncompliant` in Derived?